dfastmi/gui/dialog_view.py

```diff
@@ -0,0 +1,595 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this mainly needs to be tested

Reply:
this is the view... i think i made this so this is the only part which should be manually tested. Yesterday (during progress meeting) bert agreed on validating this. We need follow up issues for testing the dialog_ViewModel and the dialog_Model. Maybe even some more refactoring.